src/clang_utils.cc

```
@@ -57,6 +76,8 @@ optional<lsDiagnostic> BuildAndDisposeDiagnostic(CXDiagnostic diagnostic,
 
   switch (clang_getDiagnosticSeverity(diagnostic)) {
     case CXDiagnostic_Ignored:
+      assert(0);
```

Log a warning instead of asserting?

---

LLVM uses `llvm_unreachable()`, i don't know the counterpart in cquery.

---

Is this actually unreachable? Maybe we should just silently ignore the diagnostic if we get CXDiagnostic_Ignored.

---

Changed to `// llvm_unreachable`